docs/references/nextjs/clerk-middleware.mdx

```diff
@@ -430,3 +437,75 @@ export default clerkMiddleware(
 
 > [!NOTE]
 > Dynamic keys are not accessible on the client-side.
+
+### `OrganizationSyncOptions`
+
+The `organizationSyncOptions` property on the [`clerkMiddleware()`](#clerk-middleware-options) options
+object has the type `OrganizationSyncOptions`, which has the following properties:
+
+<Properties>
+  - `organizationPatterns`
+  - <code>[Pattern](#pattern)\[]</code>
+
+  Specifies URL patterns that are organization-specific, containing an organization ID or slug as a path parameter. If a request
+  matches this path, the organization identifier will be extracted and activated before rendering.
+
+  If the route also matches the `personalAccountPatterns` prop, this prop takes precedence.
+
+  Patterns must have a path parameter named either `:id` (to match a Clerk organization ID) or `:slug` (to match a Clerk organization slug).
+
+  > [!WARNING]
+  > If the organization can't be activated—either because it doesn't exist or the user lacks access—the previously active organization will remain unchanged. Components must detect this case and provide an appropriate error and/or resolution pathway, such as calling `notFound()` or displaying an [organization switcher](/docs/components/organization/organization-switcher).
```

Comment:
```suggestion
  > If the organization can't be activated—either because it doesn't exist or the user lacks access—the previously active organization will remain unchanged. Components must detect this case and provide an appropriate error and/or resolution pathway, such as calling `notFound()` or displaying an [`<OrganizationSwitcher />`](/docs/components/organization/organization-switcher).
```